Plugzy.API/Controllers/SendOtp.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.AspNetCore.Mvc;
+
+using Plugzy.Models.Request.Otp;
+using Plugzy.Service.Features.Otp.Commands;
+
+namespace Plugzy.API.Controllers;
+
+///<Summary>
+/// OTP Controller
+///</Summary>
+public class SendOtp : BaseController
```

Comment:
her işlem için ayrı controller a gerek yok,   işlemleri gruplayarak controller oluşturabilirsin.  controller ismlerinde Controller olsun mutlaka.

bu işlemi authorizecontroller a atabilirsin mesela

Reply:
Controller isimlerini hemen düzeltiyorum.
Ayrı controller olarak oluşturma sebebim görsellerdeki url uzantıları aslında. Bir klasör olarak mı gruplayayım yoksa bu iki işlemi tek bir controller altında mı gerçekleştireyim?
![post_auth](https://github.com/gkhntpc/PlugzyApp/assets/55455410/fb66e371-b372-4d0a-a07f-b6a40581df46)
![post_otp](https://github.com/gkhntpc/PlugzyApp/assets/55455410/795668ac-dcf5-4a03-bc8d-5adc742d939f)

Reply:
Commit ile çözüldü.
https://github.com/gkhntpc/PlugzyApp/pull/14/commits/43145fe4f33ace05db8e5db63a42bc62ef88fc52